app/schema.json

```diff
@@ -27,8 +27,8 @@
       "additionalProperties": false,
       "required": ["locksByViewUid", "locksDict"],
       "properties": {
-        "locksByViewUid": { "type": "object", "TODO": "more precise" },
-        "locksDict": { "type": "object", "TODO": "more precise" }
+        "locksByViewUid": { "type": "object", "additionalProperties": false },
```

Comment:
Shouldn't there be properties listed here?

Sorry, my knowledge of JSON schemas is rather limited.

Reply:
Actually, I should just delete the `additionalProperties`: This is a map from UIDs to locks, so the property names can't be validated... but we could make assertions about the locks.